### DISCUSSION TITLE
How to customize livewire power grid?

### DISCUSSION THREAD

**Discussion by Rum2001** [Power-Components/livewire-powergrid#1260] [Q&A] [answered]
I have a UserTable using Livewire power grid, and I want to customize its. Additionally, I want to add permissions for actions (@can...@endcan) for buttons. How can I achieve that ?
My php 7.4, Laravel 8, livewire 2x, livewire-powergrid 2x

**Comment by kolydart** [comment, marked as answer]
Check [custom themes.](https://livewire-powergrid.com/table/custom-theme.html)
PowerGrid ships with Tailwind and Bootstrap 5 [Themes](https://livewire-powergrid.com/get-started/configure.html#_3-choose-your-theme).
You can start customizing the default themes to create your own variation in a few steps.
